frontends/web/src/routes/settings/components/device-settings/device-name-setting.tsx

```diff
@@ -110,29 +110,20 @@ const SetDeviceNameDialog = ({ open, onClose, currentName, onInputChange, name,
       onClose={onClose}
       title={t('bitbox02Settings.deviceName.title')}
       small>
-      <div className="columnsContainer half">
-        <div className="columns half">
-          <div className="column">
-            <label>
-              {t('bitbox02Settings.deviceName.current')}
-            </label>
-            <p className="m-bottom-half">
-              {currentName}
-            </p>
-          </div>
-          <div className="column">
-            <Input
-              className={`m-none ${error && !nameIsTooShort ? nameStyle.inputError : ''}`}
-              label={t('bitbox02Settings.deviceName.input')}
-              onInput={(e) => onInputChange(e.target.value)}
-              placeholder={t('bitbox02Settings.deviceName.placeholder')}
-              value={name}
-              id="deviceName"
-            />
-            <DeviceNameErrorMessage error={error} invalidChars={invalidChars} />
-          </div>
-        </div>
-      </div>
+      <p className="m-top-none m-bottom-half">
+        {t('bitbox02Settings.deviceName.current')}
+        <br />
+        {currentName}
+      </p>
```

Comment:
Please note: this does a minor layout change with the linebreaks
<img width="494" height="334" alt="Screenshot 2025-09-10 at 15 43 27" src="https://github.com/user-attachments/assets/610315c6-94de-4e18-81ec-a47e80a03e98" />